src/main/java/az/edu/turing/model/dto/PassengerRequestDto.java

```diff
@@ -0,0 +1,42 @@
+package az.edu.turing.model.dto;
+
+import java.util.UUID;
+
+public class PassengerRequestDto {
+
+    private final String id;
+    private String name;
+    private String surname;
+
+    public PassengerRequestDto(String name, String surname) {
```

Comment:
model.request.PassengerRequestDto and model.response.PassengerResponseDto 
It would be better to create it inside the request and response packages, we will have other requests and responses.